zerobuf/render/lookOut.fbs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2016, Human Brain Project
+//                     Grigori.Chevtchenko@epfl.ch
+//
+namespace zerobuf.render;
+
+// This event is named "LookOut" instead of "Camera" because
+// it will be used in a cave environment. If you want to 
```

Comment:
it may be used in immersive environments. The orientation of a display surface or  thehead position of the observer should not be part of this event, i.e., ...magic carpet text...